content/cloud-computing/concepts/iaas/iaas.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'IaaS'
+Description: 'Infrastructure as a service (IaaS) is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Infrastructure as a service (IaaS)** is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand. These services are typically offered on a pay-as-you-go basis. The user is given control of virtual machines that are guests on a cloud server running a hypervisor such as Oracle VirtualBox or VMware ESXi. They are allowed to run arbitrary software on these virtual machines, including operating systems.
```

Comment:
Several terms here are not adequately explained. Keep in mind readers might not be familiar with terms such as pay as you go, virtual machine or hypervisor.

Reply:
Recommend attempting to simplify

Reply:
I'd also recommend, for a future PR, we make new General concept entries for hypervisor and virtual machine.